Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/util/GUIConstants.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Use the CONNECT license header here, and all other places it was auto-generated in this PR.